Client/Frontend/Browser/UserScriptManager.swift

```diff
@@ -199,8 +199,7 @@ class UserScriptManager {
       }
       
       // Inject SolanaWeb3Script.js
-      if WalletDebugFlags.isSolanaDappsEnabled,
-         tab.isPrivate == false,
+      if tab.isPrivate == false,
          Preferences.Wallet.WalletType(rawValue: Preferences.Wallet.defaultSolWallet.value) == .brave {
         let script = ScriptLoader.loadUserScript(named: "SolanaWeb3Script") ?? tab.walletSolProviderScripts[.solanaWeb3]
```

Comment:
Can we remove this so it does not default to the brave core one which is insecure :)

If the script can't be loaded, don't inject anything.

Reply:
Yup, I can remove the fallback here 👍

Reply:
Fixed in [8d7d695](https://github.com/brave/brave-ios/pull/6530/commits/8d7d695d90b85469dd5aa6feba8ffc7b583cd373)

Reply:
Our [Solana Web3 script is now used in Brave-Core (v1.47.151 and later)](https://github.com/brave/brave-browser/issues/27325). When we update core to v1.47, our checked-in `SolanaWeb3Script.js` will be removed and we'll restore behaviour to using the script from core. Updating this script is tracked with https://github.com/brave/brave-ios/issues/6680.